src/tailwind.css

```
@@ -802,6 +802,17 @@
 	}
 }
 
+@keyframes llamaPopupIn {
```

Stylelint (17.4.0)

[error] Expected keyframe name "llamaPopupIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)